content/zh/docs/tutorials/kubernetes-basics/cluster-intro.html

```diff
@@ -12,96 +8,118 @@
-
-				<p>Kubernetes 集群可以部署在物理机或虚拟机上。要开始使用 Kubernetes 开发，您可以使用 <a href="https://github.com/kubernetes/minikube">Minikube</a>。Minikube 是一个轻量级的 Kubernetes 实现，会在本机创建一台虚拟机，并部署一个只包含一个节点的简单集群。 Minikube 适用于 Linux, Mac OS 和 Windows 系统。Minikube CLI 提供了集群的基本引导操作，包括启动、停止、状态和删除。为了完成此基础训练，您将使用预先安装了 Minikube 的在线终端。</p>
-
-				<p>现在您已经知道 Kubernetes 是什么，让我们使用在线教程，开始我们的第一个集群!</p>
-
-			</div>
-		</div>
-		<br>
-
-		<div class="row">
-			<div class="col-md-12">
-				<a class="btn btn-lg btn-success" href="/docs/tutorials/kubernetes-basics/cluster-interactive/" role="button">启动互动教程 <span class="btn__next">›</span></a>
-			</div>
-		</div>
-
-	</main>
+          <!-- <h3>Objectives</h3> -->
+          <h3>目标</h3>
+                <!-- <ul>
+                    <li>Learn what a Kubernetes cluster is.</li>
+                    <li>Learn what Minikube is.</li>
+                    <li>Start a Kubernetes cluster using an online terminal.</li>
+                </ul> -->
+                <ul>
+                    <li>了解 Kubernetes 集群是什么。</li>
+                    <li>了解 Minikube 是什么。</li>
+                    <li>使用在线终端启动 Kubernetes 集群。</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <!-- <h3>Kubernetes Clusters</h3> -->
+                <h3>Kubernetes集群</h3>
+                <p>
+                <!-- <b>Kubernetes coordinates a highly available cluster of computers that are connected to work as a single unit.</b> The abstractions in Kubernetes allow you to deploy containerized applications to a cluster without tying them specifically to individual machines. To make use of this new model of deployment, applications need to be packaged in a way that decouples them from individual hosts: they need to be containerized. Containerized applications are more flexible and available than in past deployment models, where applications were installed directly onto specific machines as packages deeply integrated into the host. <b>Kubernetes automates the distribution and scheduling of application containers across a cluster in a more efficient way.</b> Kubernetes is an open-source platform and is production-ready. -->
+                <b>Kubernetes 是一个高可用集群，协调算力连接成一个单元工作。</b> Kubernetes 中允许您的应用抽象为容器进行部署，而无需将他们绑定在某个专门的节点上。要使用这种新的部署模式，需要将应用与主机解耦的方式打包应用：它们需要容器化。与以往的部署模式相比，容器化应用更加灵活可用，应用可以安装在指定的机器上，因为应用深度集成在主机中。</b> Kubernetes 以更有效的方式自动化跨集群分发和调度应用容器。</b> Kubernetes 是一个开源平台，可用在生产环境。
+                </p>
+                <!-- <p>A Kubernetes cluster consists of two types of resources: -->
+                <p>Kubernetes 集群包含两种资源类型：
+                    <ul>
+                        <!-- <li>The <b>Master</b> coordinates the cluster</li> -->
+                        <li> <b> Master </b> 协调集群</li>
+                        <!-- <li><b>Nodes</b> are the workers that run applications</li> -->
+                        <li><b> Node </b> 运行应用的工作节点</li>
```

Comment:
这个原文不是特别准确 

master node
worker node

都算是node。
```suggestion
                        <li><b>工作节点</b> 运行应用的工作节点</li>
```